packages/liveblocks-node/src/client.ts

```diff
@@ -705,9 +702,11 @@ export class Liveblocks {
    */
   public async initializeStorageDocument(
     roomId: string,
-    document: PlainLsonObject
+    document: LiveObject<S> | PlainLsonObject
   ): Promise<PlainLsonObject> {
```

Comment:
Actual question: I just noticed we're returning the document in plain LSON format here. This format is of course technically needed to get the Live objects over the wire, but I don't think in actual apps it's a nice format to work with. Should we not lift the level of abstraction a bit higher here, and either return:

- `void`
- `LiveObject` instance
- Simplified JSON (i.e. the equivalent of calling `storage.toImmutable()`)

All of those three seem better than returning a `PlainLsonObject`.

@CTNicholas @nimeshnayaju What do you think?

Reply:
Right! I think `LiveObject` is not particularly useful on the back end, so I'd probably skip that. I think `void` or JSON makes the most sense to me. However, bear in mind that `getStorageDocument` returns LSON by default, and we want our APIs to be consistent.

Reply:
Agreed, so the consistency argument rules out option 3, too. So… should we make this `void` instead? That, however, would be a breaking change, so maybe not change anything after all?

Reply:
Vincent's written up more ideas here: LB-891